### ISSUE TITLE
Git clone urls

### ISSUE THREAD

**Issue by ddnexus** [benkoshy/pagy-rails#7]
```
$ git clone --branch bootstrap git@github.com:benkoshy/pagy-rails.git

git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Shouldn't the URL be simply `https://github.com/benkoshy/pagy-rails.git` ?

**Comment by benkoshy** [comment]
I'll check and revert on the 24th of Jan

**Comment by benkoshy** [comment]
Hi Domizio (@ddnexus)

Apologies for the delay:  

There are two protocols which can be used to clone: SSH and https.

Can you confirm the the following does not work:

1. https: `git clone https://github.com/benkoshy/pagy-rails.git`
2. ssh: `git clone git@github.com:benkoshy/pagy-rails.git`

If ssh does not work, then perhaps this might be the solution? 

https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

If you could confirm on your end that would be appreciated.

**Comment by ddnexus** [comment]
I always use https because I use different and new created virtual machines to clone and other repos, so I never need ssh for virtual tries.

Admittedly my question was not clear: let me rephrase it. 

Do you really want to impose the use of ssh to a user just to clone a public demo? Shouldn't that be as accessible as possible? That is a nuisance in my case because I had to manually do it with https.

However if you think that someone would find ssh more desirable, maybe you could write both.

**Comment by benkoshy** [comment]
Both should work.  Thx for clarifying. Will get to the bottom of this.

**Comment by benkoshy** [comment]
`git clone https://github.com/benkoshy/pagy-rails.git`

I tried this one three different computers, linux, windows x2 and could not reproduce the issue: all them seemed to work fine with https.

**Comment by ddnexus** [comment]
The issue was ssh, not https. 🙂
